tests/client-dbus/Makefile

```diff
@@ -1,8 +1,13 @@
 TOX=tox
-
 .PHONY: lint
 lint:
 	$(TOX) -c tox.ini -e lint
 
 dbus-tests:
 	py.test-3 ./tests/dbus
+
+fmt:
+	yapf --style pep8 --recursive --in-place .
```

Comment:
I think it's better if the two yapf targets specify the files that they are going to work on. I ran into trouble because I had run ``make lint``` first. Then, when I ran ```make fmt``` I got the below, because yapf tried to format the directories under .tox.

```
[mulhern@localhost client-dbus]$ make fmt
yapf --style pep8 --recursive --in-place .
Traceback (most recent call last):
  File "/usr/bin/yapf", line 11, in <module>
    sys.exit(run_main())
  File "/usr/lib/python3.6/site-packages/yapf/__init__.py", line 312, in run_main
    sys.exit(main(sys.argv))
  File "/usr/lib/python3.6/site-packages/yapf/__init__.py", line 200, in main
    verbose=args.verbose)
  File "/usr/lib/python3.6/site-packages/yapf/__init__.py", line 250, in FormatFiles
    in_place, print_diff, verify, verbose)
  File "/usr/lib/python3.6/site-packages/yapf/__init__.py", line 275, in _FormatFile
    logger=logging.warning)
  File "/usr/lib/python3.6/site-packages/yapf/yapflib/yapf_api.py", line 97, in FormatFile
    encoding, in_place)
  File "/usr/lib/python3.6/site-packages/yapf/yapflib/file_resources.py", line 96, in WriteReformattedCode
    filename, mode='w', encoding=encoding, newline='') as fd:
  File "/usr/lib/python3.6/site-packages/yapf/yapflib/py3compat.py", line 30, in open_with_encoding
    return codecs.open(filename, mode=mode, encoding=encoding)
  File "/usr/lib64/python3.6/codecs.py", line 895, in open
    file = builtins.open(filename, mode, buffering)
PermissionError: [Errno 13] Permission denied: './.tox/lint/lib/python3.6/fnmatch.py'
```

Reply:
How about I add "check.py setup.py src"?  That way we don't need to update the Makefile everytime a new files is added to src?

Reply:
Yes. We don't want to specify all the files. I think check.py, setup.py, src, and tests should do the trick.